core/src/main/java/org/elasticsearch/common/geo/GeoPoint.java

```diff
@@ -88,6 +93,23 @@ public GeoPoint resetFromIndexHash(long hash) {
         return this;
     }
 
+    // todo this is a crutch because LatLonPoint doesn't have a helper for returning .stringValue()
+    // todo remove with next release of lucene
+    public GeoPoint resetFromIndexableField(IndexableField field) {
+        if (field instanceof LatLonPoint) {
+            byte[] bytes = field.binaryValue().bytes;
```

Comment:
Since `field.binaryValue()` is a `BytesRef` it's not safe in general to assume the offset will always be 0?  I think when you decode lat/lon below you can add the `offset` in there?